src/builder/builder.go

```diff
@@ -111,6 +112,7 @@ func (b *Builder) ReadBuilderConf() {
 		{`^SERVER_STATE_DIR\s*=\s*`, &b.Statedir},
 		{`^VERSIONS_PATH\s*=\s*`, &b.Versiondir},
 		{`^YUM_CONF\s*=\s*`, &b.Yumconf},
+		{`^CONTENTURL\s*=\s*`, &b.Upstreamurl},
```

Comment:
This variable in the builder.conf does not refer to the upstream (official) URL. It is in the form of **my-build-server.mydomain/update**, which gets injected into the image (by bundle-chroot-builder) so clients automatically look at this URL when doing swupd operations. So this would just save the current mixer _update/www_ dir into the upstream_url vairable each time this ran. Currently the mixer does not do anything with this option, only the chroot builder, which is why it's not being parsed in here. 

I think the right solution here is to just copy the current .clearversion file to -> www/MIXVER/upstream_url at the end/inside of BuildUpdate(), that way it's saved before it's incremented for the next build.